packages/core/test/unit/logger.test.js

```diff
@@ -0,0 +1,39 @@
+var assert = require('chai').assert;
+
+var logger = require('../../lib/logger');
+
+describe('logger', function () {
+  function reloadLogger() {
+    var path = '../../lib/logger';
+    delete require.cache[require.resolve(path)];
+    logger = require(path);
+  }
+
+  describe('defaults', function () {
+    afterEach(function () {
+      delete process.env.AWS_XRAY_DEBUG_MODE;
+      reloadLogger();
+
+    });
+    it('Should have a default logger with no transport and level set to info', function () {
```

Comment:
This test fails if the machine from which it is run has the 'AWS_XRAY_LOG_LEVEL' environment variable set already. I recommend setting it equal to the empty string then reloading the logger as you do in other tests before the assertions.

Reply:
Good spot, I've addressed this I think.

Also I think the same scenario could have happened in the final test so explicitly unset AWS_XRAY_DEBUG_MODE there.

Reply:
Tests pass in all cases now. Thanks for the fix.